src/styles/styles.js

```diff
@@ -2839,7 +2839,8 @@ const styles = {
         borderRadius: 10,
         overflow: 'hidden',
         paddingVertical: 2,
-        flexShrink: 1,
+        flexShrink: 0,
+        maxWidth: 180,
```

Comment:
could we name this constant and add it to `variables.js`?

Reply:
sure @grgia, `badgeMaxWidth` would be fine?

Reply:
updated